unstructured/partition/msg.py

```diff
@@ -120,13 +120,13 @@ def partition_msg(
         last_modification_date = (
             get_last_modified_date_from_file(file) if date_from_file_object else None
         )
-    metadata = build_msg_metadata(
```

Comment:
This fixes a bug: elements were sharing the same metadata object.
Updating an ID for one element used to update IDs for all elements... now each element gets its own metadata